src/SourceBuild/Arcade/tools/SourceBuildArcadeTarball.targets

```diff
@@ -30,7 +30,6 @@
       SetupSelfGithubInfo;
       CloneRepoAndDependentsRecursive;
       CleanClonedSource;
-      RestoreTextOnlyPackages;
```

Comment:
This disables it for the ArcadeBuildTarball case right?  Should it only be for InitializeVMR?

Reply:
Ah, this was because `CopyTextOnlyPackages` already depends on `RestoreTextOnlyPackages`. Reverted.